src/Resources/config/api_controllers.xml

```diff
@@ -1,15 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <services>
-        <defaults public="true"/>
```

Comment:
are you sure those controllers can be private?

Reply:
People can not use current service as resouce becouse they will got:
```
The file "sonata.media.controller.api.gallery" does not exists. Make sure annotations are installed and enabled.
```
This PR will add two new public aliases which will work. Technicly it is BC-break but for something which is not work now.

Reply:
> This PR will add two new public aliases which will work. Technicly it is BC-break but for something which is not work now.

Where are those public aliases?

Reply:
NVM I found them 😅